test/verify/check-storage-stratis

```diff
@@ -705,7 +742,11 @@ class TestStorageStratisReboot(storagelib.StorageCase):
         # Check that the entry has disappeared from fstab
         self.assertEqual(m.execute("grep /run/fsys1 /etc/fstab || true"), "")
 
-    def testReboot(self):
+    @testlib.skipImage("Stratis has no legacy yet", *V1_POOL_IMAGES)
```

Comment:
Stratis has no legacy yet? What does this mean?

Reply:
It means that in that image, V1 pools are still normal and and not considered legacy.